### ISSUE TITLE
Unhandled promise rejection

### ISSUE THREAD

**Issue by b0o** [googleapis/nodejs-text-to-speech#12]
Inside the TextToSpeechClient constructor, there is a promise with an unhandled rejection case:

https://github.com/googleapis/nodejs-text-to-speech/blob/754f4ece18e0faf841a72baa6db21b24580e389b/src/v1beta1/text_to_speech_client.js#L124-L130

If there's a problem with the grpc client (e.g. keyfile path doesn't exist), node complains about this unhandled promise rejection case:

```
(node:7768) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/****/****/****/server/****.key.json'
(node:7768) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:7768) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:7768) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/****/****/****/server/****.key.json'
(node:7768) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
```

Since the specific promise location wasn't mentioned in the above error output, I verified the location by augmenting the above code with `.catch()`:
```js
        textToSpeechStub.then(
          stub =>
            function() {
              var args = Array.prototype.slice.call(arguments, 0);
              return stub[methodName].apply(stub, args);
            }
        ).catch(e => {
          console.log(`Error: ${e}`)
        }),
```

Which results in node no longer complaining about the unhandled promise rejection:

```
Error: Error: ENOENT: no such file or directory, open '/home/****/****/****/server/****.key.json'
Error: Error: ENOENT: no such file or directory, open '/home/****/****/****/server/****.key.json'
```

**Comment by kinwa91** [comment]
Hi b0o,

Thanks for creating an issue for this bug. This is indeed a problem, and the rejection can be traced back to `createStub`, which creates scaffolding for auth when `TextToSpeechClient` is initialized:

https://github.com/googleapis/nodejs-text-to-speech/blob/754f4ece18e0faf841a72baa6db21b24580e389b/src/v1beta1/text_to_speech_client.js#L112-L117

Since this library is auto-generated, I've created a `gapic-generator` [issue](https://github.com/googleapis/gapic-generator/issues/2065) pending changes to better rejection handling during initialization.

**Comment by JustinBeckwith** [comment]
Good news!  This is fixed in #189
